Adding CI groovy scripts

FaridJafarov-l opened the pull request:
We are moving the CI groovy scripts here to keep the maintenance up-to-date.
https://adc.luxoft.com/jira/browse/FORDTCN-11160

FaridJafarov-l committed:
Adding CI groovy scripts
https://adc.luxoft.com/jira/browse/FORDTCN-11160

dboltovskyi commented:
Let's create a separate folder for all these scripts, e.g. `jenkins_utils`

dboltovskyi commented:
`Cancel` and `Delete` work fine

dboltovskyi commented:
`Clone` doesn't save custom parameter values
![image](https://user-images.githubusercontent.com/22884509/129218594-f6776018-3ad0-4e08-9bd0-4284ead80ecc.png)

FaridJafarov-l committed:
Replacing the hard-coded credentials with CI parameters

FaridJafarov-l committed:
Removing the existing credentials

FaridJafarov-l committed:
- Adding the credentials lookup function in order to get the credentials from CI itself
- Adding the option to check if target view exists before creating the new one

FaridJafarov-l committed:
- Removing the debuggin option to print the target view

FaridJafarov-l committed:
- Adding the jenkins_utils folder for Jenkins groovy scripts
- Removing import groovy.xml.XmlUtil statement from jenkins_utils_rename.grovy script as it is not necessary there
- Adding view description modification part to jenkins_utils_rename.groovy script

FaridJafarov-l committed:
Fixing the view description modification part
